contracts/transfer/src/wasm/keys.rs

```diff
@@ -0,0 +1,53 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+const VD_EXEC_1_0: &'static [u8] = include_bytes!("../../../.rusk/keys/267001e9ffa9eac5ec297295006c5ef7d5b16aff094f1fd9b0f4f9fbf5cca2ad.vk");
```

Comment:
Why this is part of this PR…?

I assumed the keys for the transfer were correct in the PR that was merged in `release-0.3`, it seems also a new file. Could you clarify this?

Reply:
Because I rebased to `release-0.3` and therefore I get this updates.

This might be formatting or similar but I haven't touched it.

Reply:
It doesn't seems to be formatting because it is considered a whole new file.
I'm afraid that we have some conflict – maybe you rebase a early version first, that had this file, then it was removed by a more recent change in victor's PR, but and for some reason become untracked, and you add it back in this PR.

So my suggestion is the following:
1. Just delete it and see if everything still compile and pass. If it's the case, you're good to go.
2. If not, see if there are any part of the code that contains the keys for the transfer contracts other than this file, as I assume, and check where this file is still required and change with the newest one.
3. If none of those applies, talk with @vlopes11 and figure this out. But most likely it should be one of this two.